readme.md

```diff
@@ -72,6 +72,15 @@ Kindly ask you to raise the issue in case of any suggestions/bugs.
     java -jar docker-unittests.jar -f image-tests.yml -i openjdk:9.0.1-11
    ``` 
    ![docker image tests results](./docs/image-tests-results.png)
+
+#### Export result of tests to the xml 
+1. Add -o xml option to generate xml file
+ ```bash
+    java -jar docker-unittests.jar -f image-tests.yml -i openjdk:9.0.1-11 -o xml
+   ``` 
+   ![docker image tests results](./docs/test-result-in-xml-1.png) 
```

Comment:
Change the project version in pom.xml to `1.1.0` in order to see it on the snapshots.
The `1.0.3` is obsolete and give the different output.